lms/js_config_types.py

```diff
@@ -146,8 +146,16 @@ class User(TypedDict):
     display_name: str
 
 
+class APIOrganization(TypedDict):
+    public_id: str
```

Comment:
Can we call this `org_public_id` for consistency with how we name it when sent as a query parameter?

Reply:
On a second thought, considering it is scoped to an object called `organization`, it might be a bit redundant.